test/integration/AgentTest.js

```diff
@@ -0,0 +1,24 @@
+const test = require("tape");
+const attempt = require("../helpers/RunLevel.js");
+
+test('Agent 1: Spawn Agent', t => {
+  attempt('agent01', api => new Promise(resolve => {
+    // Queue commands to move the player inside the house.
+    for (let i = 0; i < 2; i++) {
+      api.turnRight(null, 'Player');
+      for (let j = 0; j < 5; j++) {
+        api.moveForward(null, 'Player');
+      }
+    }
+
+    // Run the level. When finished, check that the Agent has spawned.
+    api.startAttempt((success, levelModel) => {
+      t.deepEqual(levelModel.player.position, [3, 3]);
```

Comment:
Relying on agent position seems fragile.

Reply:
Good call.  I've updated this test to also verify that the Agent is not present at the start of the level, and spawns when the player completes the level.  Does this approach look ok?

Reply:
The hardcoded `[3, 3]` position is a bit fragile, but it's tied to a fixture that is checked in specifically for the integration tests.  The fixture won't change unless we intentionally update it to match any live-site level changes.